app/src/app/poll/poll.component.html

```diff
@@ -86,90 +86,93 @@ <h4 *ngIf="displayedResult !== undefined" i18n>Results of round {{ displayedResu
 
   </div>
 
-  <div class="information" *ngIf="!!answer">
-    
-    <ng-template #firstRoundTmpl>
-      <p i18n>
-        This is the first round of the poll.
-        <ng-container *ngIf="!hasCurrentRoundBallot()">
-          You can submit a first ballot, in which case you will become a participant of the poll.
-        </ng-container>
-        In the next rounds, you will be able to change your vote, based on the results of the previous round.
-        The first round will end
-        <ng-container [ngSwitch]="roundDeadlinePassed()">
-          <ng-container *ngSwitchCase="true">
-            as soon as three participants have voted.
-          </ng-container>
-          <ng-container *ngSwitchCase="false">
-            {{ answer.RoundDeadline | nearDate:'inside' }}.
-          </ng-container>
-        </ng-container>
-        <ng-container *ngIf="!hasCurrentRoundBallot()">
-          It will not be possible to join the poll once the first round has ended.
-        </ng-container>
-        Then, each round will last {{ answer.MaxRoundDuration | duration }}, or until all participants have voted,
-        whichever comes first.
-      </p>
-    </ng-template>
+  
 
-    <ng-container [ngSwitch]="answer.State">
-      <ng-container *ngSwitchCase="'Waiting'">
-        <p i18n>
-          The poll will start {{ answer.Start | nearDate:'inside' }}.
-          You won't be able to vote until that date.
-        </p>
-      </ng-container>
-      <ng-container *ngSwitchCase="'Terminated'">
+  <div class="infomenu">
+    <div class="infomenuincon">
+      <mat-icon class="helpbutton" id="helpbutton">help</mat-icon>
```

Comment:
CSS names are much easier to read when words are separated with `-`. I regret to not have done it since the beginning for the class and id names. Moreover, the information "bar" is not a menu. Hence I suggest info-panel instead of infomenu, info-panel-icon instead of infomenuincon, and help-button instead of helpbutton.